### ISSUE TITLE
Scaffold out protocol thoughts / initial design

### ISSUE THREAD

**Issue by jdogcoder** [thepurplebubble/protocol#1]

**Comment by theamazing0** [comment]
Reposting my opinion from slack for accessibility,

> I think a new protocol should be made would be if it had some sort of design advantage of matrix

> Purple could mean matrix, some other color could mean a p2p protocol, some other could mean a compatibility feature with insecure protocols
